app/controllers/actuals/uploads_controller.rb

```diff
@@ -55,6 +55,8 @@ def update
         @success = true
         flash.now[:notice] = t("action.actual.upload.success")
       end
+
+      @has_zero_and_blank_error = importer.errors.find { |error| error.value == "0, blank" }.present?
```

Comment:
Does this _have_ to be conditional on this specific error?

It seems like it's a helpful message if there are _any_ errors.

If it absolutely does have to be conditional on this, I'm not sure if I feel good about searching for that specific string

Reply:
I did wonder, but I think it makes sense in that those are the default values when you download the template. I.e. if they changed the values from the default of zero + blank, it's likely they aren't _solely_ attempting to add a comment (without adding/changing values). In that case, they'd fix the value errors and then the comment would be added as usual, when the values are valid. I can go back to Ming though if you think it should appear in other cases?

Reply:
Ah, I see! 

My thinking was that showing it on any would would remove a decent amount of complexity, and that possibly brittle string comparison

Reading the message again, seems like it's always useful to me. Maybe pushing back might be worth it.
```
If you want to upload a comment without a financial amount, please do not provide invalid values via bulk upload. You can go to the Comments section of the relevant activities and create a comment there.
```

Not sure what @CristinaRO thinks, I was a bit slower reviewing this!

Reply:
If it was for me, I'd add the signposting unconditionally. (I don't think it would be too confusing for users who are encountering other errors.) Is it worth to check if BEIS agrees?

Reply:
Yep, and I'll let Ming know